.github/CONTRIBUTING.md

```
@@ -51,6 +51,12 @@ While developing you can run a single test group and use [grep](https://mochajs.
 npm test -- --grep=lifecycle
 ```
 
+To run the whole suite quickly (without coverage), `npm run test:parallel` fans the tests across your CPU cores:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)